src/applications/vaos/new-appointment/components/VAFacilityPage/FacilitiesRadioWidget.jsx

```diff
@@ -10,6 +10,7 @@ const INITIAL_FACILITY_DISPLAY_COUNT = 5;
  * form system.
  */
 export default function FacilitiesRadioWidget({
+  id,
```

Comment:
It's probably worth checking for this issue in a unit test. If you had a test that displayed a couple facilities on this page, you could check that they all had the same `name` attribute.